tests/e2e/test_uuid_passthrough_e2e.sh

```
@@ -0,0 +1,211 @@
+img.save(buf, format="PNG")
+print(base64.b64encode(buf.getvalue()).decode(), end="")
+')"
+IMG_A_UUID="image-a"
+IMG_B_UUID="image-b"
+
+cleanup() {
+    pkill -f "dynamo.frontend" 2>/dev/null || true
+    pkill -f "dynamo.vllm" 2>/dev/null || true
+    pkill -f "VLLM" 2>/dev/null || true
+    sleep 2
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)